livre/core/CMakeLists.txt

```diff
@@ -16,7 +17,6 @@ set(LIVRECORE_PUBLIC_HEADERS
   render/GLSLShaders.h
   util/Utilities.h
   lunchboxTypes.h
-  mathTypes.h
```

Comment:
I want to keep this separated. Maybe types.h can be removed from mathTypes.h and that will be more useful.